tests/system/action/projector/test_toggle.py

```diff
@@ -86,6 +86,27 @@ def test_correct_add_projection(self) -> None:
         projector = self.get_model("projector/23")
         assert projector.get("current_projection_ids") == [1]
 
+    def test_toggle_unstable_move_into_history(self) -> None:
+        self.setup_models(False)
+        self.set_models(
```

Comment:
set the scoll to something different than 0, so you actually assert something below.

Reply:
Set the scroll before the action to 100.